app/views/shared/_donation_platforms.html.haml

```diff
@@ -1,33 +1,34 @@
-.row
```

Comment:
@KimberleyCook this partial is already wrapped in a .container where we are importing it, I don't think we should add it here.